bpf/bpf_lxc.c

```diff
@@ -196,7 +196,9 @@ static __always_inline int ipv6_l3_from_lxc(struct __ctx_buff *ctx,
 		if (info != NULL && info->sec_label) {
 			*dst_id = info->sec_label;
 			tunnel_endpoint = info->tunnel_endpoint;
+#ifdef ENABLE_IPSEC
 			encrypt_key = get_min_encrypt_key(info->key);
+#endif
```

Comment:
Should we move the preprocessor guard inside `get_min_encrypt_key` to avoid having too many preprocessor directives in this function/file? We already have many and it doesn't help readability IMO.

Reply:
Good point, will move the preprocessor guard into `get_min_encrypt_key`.